base_layer/wallet_ffi/tests/test_seed_peer_validation.py

```
@@ -0,0 +1,160 @@
+Test Seed Peer Discovery Validation
+
+Focused test for validating that FFI seed peers work correctly with nextnet
+"""
+
+import sys
+import os
+
+# Add the Python module path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'python'))
+
+import tari_wallet as tw
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)